proposals/networkservicemesh.adoc

```diff
@@ -0,0 +1,95 @@
+
+*Statement on alignment with CNCF mission*:
+
+Network Service Mesh is designed to allow a workload to dynamically request at any point in its lifecycle the L2/L3 Network Service it needs.   It makes networking itself Cloud native by loosely coupling with the existing immutable infrastrcture in K8s to deliver whatever non-standard Networking needs a workload has with minimal toil.
+
+The Network Service Mesh model is:
+
+* Developers ask for what Network Service they want by name
+* Developers can add metadata to their request for a Network Service with labels.
+* Developers simply get the Network Service they asked for
+
+Developers never have to know or think about 35 year old legacy Networking concepts like:
+
+* Subnets
+* Interfaces
+* Routes
+* IPAM
+
+*Sponsor / Advisor from TOC*: Joe Beda, Matt Klein
+
+*Unique identifier:* networkservicemesh
+
+*Prefered Maturity Level*: Sandbox
+
+*License*: Apache License v2.0
+
+*Source control repositories*: link:https://github.com/networkservicemesh[GitHub]
+
+*Initial Committers*:
+
+* Frederick Kautz (doc.ai)
+* Nikolay Nikolaev (VMWare)
+* Ed Warnicke (Cisco)
+
+*Infrastructure requirements (CI / CNCF Cluster)*: 
+
+* CI (currently use CircleCI)
+* Packet/GKE/AKS/EKS/etc time for system tests
+
+*Communication Channels*:
+
+* Mailing List: https://groups.google.com/forum/#!forum/networkservicemesh
+* IRC: #networkservicemesh on Freenode
+* Weekly Meetings: 
+  - link:https://docs.google.com/document/d/1C9NKjo0PWNWypROEO9-Y6haw5h9Xmurvl14SXpciz2Y/edit[Main Meeting]
+  - link:https://docs.google.com/document/d/1113nzdL-DcDAWT3963IsS9LeekgXLTgGebxPO7ZnJaA/edit[Documentation subgroup meeting]
+  - link:https://docs.google.com/document/d/1L6kwLW6yvj1EfUgDbdurCpgw2qWrX3ZJJ8Vh3FRqrOo/edit[Use Case subgroup meeting]
+
+*Issue Tracker*: link:https://github.com/networkservicemesh/networkservicemesh/issues[Github Issues]
+
+*Website*: https://networkservicemesh.io/
+
+*Release Methodology and Mechanics*: link:https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/spec/release.md[Release Mechanics Defined in Github]
+
+*Social Media Accounts*:
+
+  * link:https://www.linkedin.com/company/networkservicemesh[LinkedIn]
+  * link:https://twitter.com/nservicemesh[Twitter]
+
+*Community size and existing sponsorship*:
+
+* Github Stars: 111
+* Github Forks: 38
+* Code Contributors: 18
+* Usual meeting census: ~20
+* Youtube views of Weekly meetings: link:https://www.youtube.com/results?search_query=network+service+mesh+wg+2019[More than 100 typically]
+* Youtube views of Network Service Mesh Intro talk at Kubecon NA 2018: link:https://www.youtube.com/watch?v=YeAKtUFaqQ0&t=2s[More than 1400]
+* Existing Sponsors:
+  * Bell Canada
+  * Cisco
+  * doc.ai
+  * Orange
+  * RedHat
+  * VMWare
```

Comment:
Lumina Networks

Reply:
@gpremsankar Can you +1 this to confirm?

Reply:
> Lumina Networks

+1